### DISCUSSION TITLE
Does the elasticsearch-logger plugin support multiple elasticsearch nodes?

### DISCUSSION THREAD

**Discussion by xiaoxuanzi** [apache/apisix#8431] [Q&A]
The endpoint_addr attribute type of elasticsearch-logger is a string, it seems that multiple elasticsearch nodes are not supported.

I want to know why endpoint_addr is designed as a string instead of an array.

Thanks.

For example, The Python Elasticsearch client supports sending API requests to multiple nodes in the cluster.

```
from elasticsearch import Elasticsearch

NODES = [
    "https://localhost:9200",
    "https://localhost:9201",
    "https://localhost:9202",
]

ELASTIC_PASSWORD = "<password>"

client = Elasticsearch(
    NODES,
    ca_certs="/path/to/http_ca.crt",
    basic_auth=("elastic", ELASTIC_PASSWORD)
)
```

**Comment by tzssangglass** [comment]
> I want to know why endpoint_addr is designed as a string instead of an array.

The initial version is designed to support only a single node.

I would like to know what is the benefit of supporting multiple nodes? Will it reduce the pressure on the es cluster to write data?

**Reply by xiaoxuanzi** [reply]
Hi @tzssangglass 
Thank you very much for your reply.
I want to know, if the only node is down, how to deal with this situation.

**Reply by tzssangglass** [reply]
> I want to know, if the only node is down, how to deal with this situation.

will have this problem. PR is welcome!
